server/controllers/resourceParserController.js

```diff
@@ -6,16 +7,43 @@ const resourceParserController = {
   parsePut: async (req, res, next) => {
     
     try{
-
-        
-      res.locals.newResource = newResource;
-      return next();
+      
+      const msgObject = await req.body[0];
```

Comment:
Just a reminder to address what @sflam2013 pointed out in discord, that this can just be req.body now that you're not using the sample-objects file.